pkg/cmd/service/service_test.go

```diff
@@ -311,7 +302,7 @@ func TestBuildServiceWithPresharedKeyAuthentication(t *testing.T) {
 }
 
 func TestHTTPServerWithCORS(t *testing.T) {
-	config, err := GetServiceConfig()
+	config, err := DefaultConfigWithRandomPorts()
```

Comment:
@craigpastro haha I changed my mind on this. I like using `GetServiceConfig` because it's higher fidelity and the actual path our server uses on startup to get config, but we'd always be overrding the GRPC and HTTP addresses based on the auto assigned port, and so since we don't use env variables or anything in our tests today I decided I changed my mind on this point.

Sorry for the confusion :)

Reply:
lol 👍